README.md

```diff
@@ -6,6 +6,13 @@
 lightweight RTOS-based platform to support low-end IoT devices.  
 Please find project details like **APIs**, **Specification** and **Long-term Goals** on our [Tizen Site](https://source.tizen.org/documentation/tizen-rt).
 
+## Contents
+
+> [Quick Start](#quick-start)  
```

Comment:
@sunghan-chang How about indexing with numbers for list of contents, looks better I think.
1.Quick Start
2.Supported Board
...